exporter/exporterhelper/metadata.yaml

```diff
@@ -9,79 +9,79 @@ status:
 
 telemetry:
   metrics:
-    exporter_sent_spans:
```

Comment:
if we are breaking compat by changing the name, shouldn't we also be switching from underscores to dots?

Reply:
we're not going to break compat. the `_` is the separator that metrics are emitted with today via prometheus. i'd like to keep that consistent for metrics emitted via OTLP as well